Essentials/src/messages.properties

```diff
@@ -61,6 +61,7 @@ cleaning=Cleaning userfiles.
 clearInventoryConfirmToggleOff=\u00a76You will no longer be prompted to confirm inventory clears.
 clearInventoryConfirmToggleOn=\u00a76You will now be prompted to confirm inventory clears.
 commandCooldown=\u00a7cYou cannot type that command for {0}.
+commandDisabled=\u00a7cThis command\u00a76 {0}\u00a7c is disabled.
```

Comment:
This needs to be either "This command is disabled." or "The command {0} is disabled."